product_maintenance_equipment_link/views/maintenance_equipment_views.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="maintenance_equipment_form_view" model="ir.ui.view">
+        <field name="name">maintenance.equipment.form.view (in product_maintenance_equipment_link)</field>
+        <field name="model">maintenance.equipment</field>
+        <field name="inherit_id" ref="maintenance.hr_equipment_view_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//div[@name='button_box']" position="inside">
+                <button name="action_view_products" type="object" class="oe_stat_button" icon="fa-list">
+                    <field string="Products marked for this equipment" name="count_product_ids" widget="statinfo" />
```

Comment:
Replace current too long legend by a shorter one, as smart button available space is very narrow:

![imagen](https://github.com/user-attachments/assets/b632d52c-8bde-49c7-b189-5099d5ac8599)